testing/integration/upgrade_test.go

```diff
@@ -261,7 +261,7 @@ func TestStandaloneUpgradeWithGPGFallback(t *testing.T) {
 	testStandaloneUpgrade(ctx, t, agentFixture, parsedVersion, toVersion, "", false, false, true, customPGP)
 }
 
-func TestStandaloneUpgradeToSpecificSnapshotBuild(t *testing.T) {
+func TestStandaloneDowngradeToPreviousSnapshotBuild(t *testing.T) {
```

Comment:
I would keep the previous test name as it's the feature name. we may add a comment for clarity

```suggestion
// Here we test the ability of elastic agent to upgrade to a given snapshot build (not necessarily the latest)
// by passing a version `x.y.z-SNAPSHOT+<buildid>`. In order to test this, we look for a build for the current
// snapshot version other than the latest, downgrade to such version and verify that we get the correct
// git commit hash for the specified build (to detect that we didn't install the latest snapshot build by mistake)
func TestStandaloneDowngradeToSpecificSnapshotBuild(t *testing.T) {
```